ui/search/nuxeo-results-view.js

```diff
@@ -0,0 +1,447 @@
+        /**
+         * The headers passed on to `provider`.
+         * Already set by default are 'fetch-document': 'properties' and 'translate-directoryEntry': 'label'.
+         */
+        headers: {
+          type: String,
+          value: { 'fetch-document': 'properties', 'translate-directoryEntry': 'label' },
+        },
+        /**
+         * List of comma separated values of the document schemas to be returned.
+         * All document schemas are returned by default.
+         */
+        schemas: {
+          type: String,
+        },
+        /**
+         * If `true`, requests from `provider` are in flight.
+         */
+        loading: {
+          type: Boolean,
+          reflectToAttribute: true,
+          value: false,
+        },
+        /**
+         * The name of the search layout.
+         */
+        searchName: String,
+        /**
+         * The aggregations returned by `provider`.
+         */
+        aggregations: {
+          type: Object,
+          observer: '_aggregationsChanged',
+          notify: true,
+        },
+        /**
+         * Name of the quick filters to be displayed in case you don't want to display all of them.
+         * Expected format : ['quickfilter1','quickfilter2'].
+         */
+        quickFilters: {
+          type: Array,
+          notify: true,
+        },
+        /**
+         * If `true`, the current element is visible.
+         */
+        visible: {
+          type: Boolean,
+          value: false,
+        },
+        /**
+         * If `true`, automatically execute the search when either `provider` or `params` changes.
+         */
+        auto: {
+          type: Boolean,
+          value: false,
+        },
+        /**
+         * If `true`, display the top filtering panel.
+         */
+        showFilters: {
+          type: Boolean,
+          value: false,
+        },
+        /**
+         * If `true`, opens the collapsible top filtering panel.
+         */
+        opened: {
+          type: Boolean,
+          value: false,
+        },
+        /**
+         * An external search form (containing its own page provider) will be used, instead of the embeded one.
+         * In case this is set, the following properties should be ignored, as the behavior they affect will be
+         * controlled by the search form: `provider`, `pageSize`, `params`, `schemas`, `enrichers`, `headers`,
+         * `loading`, `aggregations`, `quickFilters` and `auto`.
+         */
+        searchForm: {
+          type: Object,
+          value: null,
+          observer: '_searchFormChanged',
+        },
+        /**
+         * If `true`, aggregates from page provider definition will not be computed.
+         */
+        skipAggregates: Boolean,
+        _params: Object,
+        _paramsCount: Number,
+        _nxProvider: HTMLElement,
+        _hideCounter: {
+          type: String,
+          computed: '_computeHideCounter(opened, _params.*)',
+          value: 'hidden',
```

Comment:
I don't think a value applies for computed properties? 🤔

Reply:
Couldn't find a reason to have this in the Polymer docs. Should we remove it?

Reply:
I'm in favor of removing it.